Kopernicus/Kopernicus/RuntimeUtility/RnDFixer.cs

```diff
@@ -252,9 +252,10 @@ void Start()
                     }
                 }
 
-                if (planetItem?.planet?.GetComponent<StarComponent>() != null)
+                // Kill Rotation
+                if (body.Has("RDRotation") ? !body.Get<bool>("RDRotation") : planetItem?.planet?.GetComponent<StarComponent>() != null)
```

Comment:
Tbh, I would just check if the body is in KopernicusStar.Stars, but this works too.

Reply:
I wasn't sure how KopernicusStar worked

do you keep all stars in there? even those with "GivesOffLight = False" ?

Reply:
Yes

Reply:
honestly I would restrict the "off" feature only to those stars that have coronas, since the only downside of rotating stars is that the corona rotates together with the star >_>

Reply:
Thats fine.